website/docs/plugin/framework/handling-data/attributes/number.mdx

```diff
@@ -0,0 +1,126 @@
+---
+page_title: 'Plugin Development - Framework: Number Attribute'
+description: >-
+  Learn the number attribute type in the provider development framework.
+---
+
+# Number Attribute
+
+<Tip>
+
+Use [Float64 Attribute](/terraform/plugin/framework/handling-data/attributes/float64) for 64-bit floating point numbers. Use [Int64 Attribute](/terraform/plugin/framework/handling-data/attributes/int64) for 64-bit integer numbers.
+
+</Tip>
+
+Number attributes store an arbitrary precision (generally over 64-bit, up to 512-bit) number. Values are represented by a [number type](/terraform/plugin/framework/handling-data/types/number) in the framework.
+
+In this Terraform configuration example, an number attribute named `example_attribute` is set to the value `1.23`:
+
+```terraform
+resource "examplecloud_thing" "example" {
+  example_attribute = 1.23
+}
+```
```

Comment:
Nit: Do we want to consider using an example where the required precision is > 64 bit to emphasise the use-case?

Reply:
Great suggestion -- let me find a fun one that Terraform likes.